spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/oauth/DataflowOAuthIT.java

```diff
@@ -54,12 +56,34 @@ public void testSecuredSetup() throws Exception {
 				.ignoreExceptions()
 				.atMost(120, TimeUnit.SECONDS)
 				.until(() -> {
-					log.debug("Checking auth using curl");
+					log.info("Checking auth using curl");
 					ExecResult cmdResult = execInToolsContainer("curl", "-u", "janne:janne", "http://dataflow:9393/about");
 					String response = cmdResult.getStdout();
 					log.debug("Response is {}", response);
 					boolean ok = response.contains("\"authenticated\":true") && response.contains("\"username\":\"janne\"");
 					log.info("Check for oauth {}", ok);
+					if (ok) {
+						try {
+							String version = JsonPath.parse(response).read("$.versionInfo.core.version");
+							log.info("Version=[{}]", version);
+							String api = "tasks/executions";
+							if (VersionUtils.isDataFlowServerVersionGreaterThanOrEqualToRequiredVersion(
+									VersionUtils.getThreePartVersion(version), "2.11.3")) {
+								api = "tasks/thinexecutions";
+							}
+							log.info("Checking {}", api);
+							cmdResult = execInToolsContainer("curl", "-u", "janne:janne", "http://dataflow:9393/" + api);
+							response = cmdResult.getStdout();
+							log.debug("Response is {}", response);
+							ok = !JsonPath.parse(response).read("$._links.self.href", String.class).isEmpty();
+
+							// TODO add checks for new endpoints to check security
```

Comment:
This does add coverage for the `thin/executions` endpoint but what is the expectation for newly added endpoints? How will we know to abide by this comment? 

The issue that we ran into was that PRO was out of sync w/ OSS endpoint mappings. This does not fill that gap.

Reply:
This test is executed as part of ci-it-security.
The Pro will be covered by the endpoint test running in CF ATs.

Reply:
> The Pro will be covered by the endpoint test running in CF ATs.

Which endpoint test? The point of https://github.com/pivotal/scdf-pro/issues/192 is to find out issues prior to CF ATs (in the PRO tests).

Reply:
I added ci-it-security to scdf-pro to perform similar checks on added endpoints to ensure they are protected.